sdk/python/feast/base_feature_view.py

```diff
@@ -64,7 +64,7 @@ def __init__(
 
         Args:
             name: The unique name of the base feature view.
-            features: The list of features defined as part of this base feature view.
+            features (optional): The list of features defined as part of this base feature view.
```

Comment:
prefer `list of features` since `features` here actually refers to features; e.g. I would consider `schema` a list of fields since it contains not just features but also join keys, but `features` I consider a list of features